### PR TITLE
Added unit test to ping /logout endpoint

### PR DESCRIPTION
**Pull request by trevorjr31**
### What this PR does (required):
- Test for successful response from logout endpoint
- Addresses Ticket 3
#3  

### Any information needed to test this feature (required):
- run `npm test` from the server directory to run the suite

### Any issues with the current functionality (optional):
N/A

### CODE REVIEW

File: server/test/logout.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      chai
+        .request(app.server)
+        .get(`/auth/logout`)
+        .end((err, res) => {
+          res.should.have.status(200);
+          done();
+        });
+    });
+  });
+});
```
Comment (ezeilo-su) [inline server/test/logout.spec.js:20]: :+1:

**Review by ezeilo-su — COMMENTED** [review]
This one is good!

File: server/test/logout.spec.js (hunk 1 above)
Comment (trevorjr31) [reply server/test/logout.spec.js:20]: 😄

**Comment by trevorjr31** [comment]
Added Link! @ezeilo-su Thank you

File: server/test/logout.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("Logout Test", () => {
+  describe("GET /logout", () => {
+    it("Should successfully receive log out response", (done) => {
+      chai
+        .request(app.server)
+        .get(`/auth/logout`)
```
Comment (rajivtitus) [inline server/test/logout.spec.js:13]: Just so we are aware, GET isn't the verb that makes the most sense for a /logout path! (see this [article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!

File: server/test/logout.spec.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        .end((err, res) => {
+          res.should.have.status(200);
```
Comment (rajivtitus) [inline server/test/logout.spec.js:15]: Please check the response text here as well

**Review by rajivtitus — COMMENTED** [review]
Good work! Just a minor comment, thanks!

File: server/test/logout.spec.js (hunk 2 above)
Comment (trevorjr31) [reply server/test/logout.spec.js:13]: I would usually use POST in this logout situation. I did put POST in first for the test and then had to go back and check the implementation. I changed it to GET for the test to run correctly. I see why it is important to separate QA and development in this sense.

File: server/test/logout.spec.js (hunk 3 above)
Comment (trevorjr31) [reply server/test/logout.spec.js:15]: Added the check. Thank you.

**Review by rajivtitus — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**